front_end/object_ui/CustomPreviewComponent.js

```diff
@@ -30,8 +30,8 @@ ObjectUI.CustomPreviewSection = class {
 
     if (customPreview.hasBody) {
       this._header.classList.add('custom-expandable-section-header');
-      this._header.addEventListener('click', this._onClick.bind(this), false);
       this._expandIcon = UI.Icon.create('smallicon-triangle-right', 'custom-expand-icon');
+      this._expandIcon.addEventListener('click', this._onClick.bind(this), false);
```

Comment:
This should probably have a comment indicating that it's for DDT, and explaining why it's done. The first so that it's easy to spot in merges. I assume it's so we don't intercept clicks on the URL, but we should make that obvious.

Reply:
Good point. Added.